### ISSUE TITLE
Improvement: Add a background image for the bubble as an alternative of bubbleColor

### ISSUE THREAD

**Issue by LouisBorlee** [andreamazz/BubbleTransition#42]
Hello,

First of all, thank you for sharing your work, it's great!

I would like to know if you ever considered adding an optional UIImage to be used as a bubble background?

It would be a nice improvement I feel.

Cordially,

**Comment by andreamazz** [comment]
Wouldn't it look weird, given the massive differences in ratio and sizes?

**Comment by LouisBorlee** [comment]
I suppose it would depend on how you put it.

It may be possible to use an aspectFill ratio on a large-scaled image. 

My use case is: 
I have a [floating circle button](https://user-images.githubusercontent.com/1114435/37475683-8b700222-2873-11e8-8bee-080938620528.png) which I would like to extend on click.
It would show some other colored blocks.

**Comment by andreamazz** [comment]
I see. It should be doable, maybe adding the ability to fade it out and in too.

**Comment by LouisBorlee** [comment]
Indeed! Even capture a screen-sized image to add as the receiver viewController's background maybe.
